app/components/certificate/description.tsx

```diff
@@ -17,7 +17,7 @@ export default function DescriptionSection({
     <Flex
       flexDirection="column"
       gap="3"
-      marginTop={{ base: '5', md: '14' }}
+      marginTop="5"
```

Comment:
Did you mean to include this?

Reply:
yep I noticed there's no need to have a large margin there so I went with the base as default now